packages/bappo-components/src/primitives/View/View.native.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import RN from 'react-native';
+
+import { ViewProps } from './types';
+
+const View = React.forwardRef(
+  (
+    {
+      accessibilityLabel,
+      children,
+      className,
+      onLayout,
+      pointerEvents,
+      style,
+      testID,
+    }: ViewProps,
+    ref,
+  ) => {
+    const nativeViewRef = React.useRef<RN.View>(null);
+
+    React.useImperativeHandle(ref, () => ({
+      // To be able to use View inside TouchableHighlight/TouchableOpacity
+      setNativeProps: (...args: Array<any>) =>
+        nativeViewRef &&
+        nativeViewRef.current &&
+        nativeViewRef.current.setNativeProps({ ...args }),
```

Comment:
```suggestion
        nativeViewRef.current.setNativeProps(...args),
```

Reply:
Hi William, when I make the suggested change I get the following error:

![image](https://user-images.githubusercontent.com/70428743/101348055-b56b7b00-3893-11eb-90f2-f296b2584cb3.png)

Do I maybe have to change it to?:

![image](https://user-images.githubusercontent.com/70428743/101348218-f794bc80-3893-11eb-9c3b-3732283221f0.png)

Thanks!

Reply:
Hi Emil. That looks good.

Reply:
Hi William, thanks, I've applied the change to `setNativeProps()`.